src/config.rs

```diff
@@ -513,6 +513,11 @@ impl Context {
                 );
                 self.sql.set_raw_config(key.as_ref(), value).await?;
             }
+            Config::Addr => {
+                self.sql
+                    .set_raw_config(key.as_ref(), value.map(|s| s.to_lowercase()).as_deref())
+                    .await?;
```

Comment:
ppl may complain that they want sth. as Prename.Familyname@sth but we'll see. the issues with different cases are way larger than cosmetic case (also, it probaobly does not want anyways for these ppl to maintain case across instances)

Reply:
just in case you do not know the variation of the old "smokie" song: https://www.youtube.com/watch?v=CsrfovOPcjk#t=0m51s